spec/index.html

```diff
@@ -6327,9 +6329,9 @@ <h5>hasLANG</h5>
             <pre class="prototype nohighlight"> <span class="return">xsd:string</span>  <span class="operator">hasLANG</span> (<span class="type">RDF term</span> <span class="name">term</span>)
             </pre>
             <p>
-              The function `hasLANG` returns <code>true</code> if the
-              argument is a literal with a language
-              tag. Otherwise, the function returns <code>false</code>.
+              Returns `true` if the RDF term argument is a literal with a 
```

Comment:
It seems somewhat inconsistent to me that the argument of `LANG` and of `LANGDIR` is explicitly a literal whereas the argument of this function (and the next) is an RDF term. Do you remember whether there was a reason for this difference?

Reply:
The `is*` functions have a signature of `(RDF term)`. 

These two (`hasLANG`, `hasLANGDIR`) are similar. Earlier, they were `is*` but I think that `has*` reads better for these accessors because it is not "is this term a language?", it is "does this term have a language component?" and the question can be extends to any RDF term.

(Un-)language `""` is sort of special (due in part to XML). We can use that to separate "" and error. 

`LANG(<notALiteral>)` returning `""` does not make sense and should be an error. "" (no language tag : literals _may_ have a language tag) and error (it's not a literal -- wrong question) can be different.

Reply:
Thanks for the explanation. Makes sense.